f/Form-submission-and-saving-data-to-the-database: Form saving to dat…

seanlaing opened the pull request:
### Link to Ticket
[http://grails.asia/grails-tutorial-for-beginners-form-submission-and-saving-data-to-the-database](http://grails.asia/grails-tutorial-for-beginners-form-submission-and-saving-data-to-the-database)

### Summary
- Created a submission form
- Connect it to a controller
- Setup save method in controller
- Connected all the parts so a gsp form will save a person data using a PersonController save method


### Reasonable Test Coverage
- [ ] YES

### Migrations
- [ ] YES
- [ ] NO

### Please validate the following functionality before reviewing code
- [ ] Check the following page, http://localhost:8080/GrailsPractice/person/form
- [] Success message when data is submitted

### Deploy Notes

## Areas of the application that may be impacted by this change
A new person controller was created as well as form gsp.

## Notes, links to resources, etc

Commit:
f/Form-submission-and-saving-data-to-the-database: Form saving to database
- Created a submission form
- Connect it to a controller
- Setup save method in controller
- Connected all the parts so a gsp form will save a person data using a PersonController save method

ctwplyp reviewed (changes requested):
Looks like you forgot to commit the PersonController and equivalent testing

Commit:
f/Form-submission-and-saving-data-to-the-database:  Added missing files per pull request
- See pull request comments, https://github.com/ctwplyp/GrailsPractice/pull/7#pullrequestreview-86499376

Commit:
f/Form-submission-and-saving-data-to-the-database: Renamed variables correctly
